plugins/fritz-box7490/de/fritzbox7490.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
```

Comment:
Please create a separate pull request for this new plugin.

Regarding the path: the "de" path item serves no purpose, I guess.
Please remove the file extension.
Please rename the plugin in order to specify its purpose more clearly:
* does it apply only to the one model specified? If yes: please mention this in the documentation header.
* does it cover _all_ aspects of the router? if no: please add this to the plugin name (e.g. "fritzbox_traffic").

Please change the documentation header into the perldoc format.